public/Day 108/script.js

```
@@ -1,173 +1,334 @@
+  // Update UI
+  uploadPlaceholder.classList.add("hidden");
+  canvasWrapper.classList.remove("hidden");
+  enableControls(true);
+}
+
+function enableControls(enable) {
+  const tools = [
+    grayscaleBtn, edgeBtn, cartoonBtn, removeBgBtn, sepiaBtn, invertBtn, sharpenBtn,
+    resetBtn, downloadBtn, compareBtn, brightnessSlider, contrastSlider
+  ];
+  tools.forEach(btn => btn.disabled = !enable);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)